packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -55,13 +55,13 @@ const configurations = {
   },
   DEVNET: {
     committeeName: 'Economic Committee',
-    // TODO: Update the addresses after confirmation
     voterAddresses: {
       gov1: 'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce',
       gov2: 'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang',
+      gov3: 'agoric1f0h5zgxyg3euxsqzs0506uj4cmu56y30pqx46s',
     },
     highPrioritySendersConfig: {
-      addressesToAdd: [],
+      addressesToAdd: ['agoric1f0h5zgxyg3euxsqzs0506uj4cmu56y30pqx46s'],
       addressesToRemove: ['agoric1w8wktaur4zf8qmmtn3n7x3r0jhsjkjntcm3u6h'],
```

Comment:
how do we remove `...u6h`? I don't see it listed in `voterAddresses`.

Reply:
The `voterAddresses` map includes the addresses that will be invited to join the new committee and charter. The lists `addressesToAdd` and `addressesToRemove` are linked to `highPrioritySenders`. The new member `...x46s`, has been added to `addressesToAdd`. An outgoing member from the previous committee, .`..u6h`, has been placed in `addressesToRemove`.

Reply:
ok... I recognize `...u6h` as gov3 now, and I see it's in the [devnet highPrioritySenders](https://vstorage.agoric.net/?path=highPrioritySenders&endpoint=https%3A%2F%2Fdevnet.rpc.agoric.net%3A443&height=null).